cudamapper/src/overlapper.cpp

```diff
@@ -136,11 +136,11 @@ void Overlapper::post_process_overlaps(std::vector<Overlap>& overlaps, const boo
 {
     const auto num_overlaps = get_size(overlaps);
     bool in_fuse            = false;
-    int fused_target_start;
-    int fused_query_start;
-    int fused_target_end;
-    int fused_query_end;
-    int num_residues = 0;
+    int fused_target_start  = 0;
+    int fused_query_start   = 0;
+    int fused_target_end    = 0;
+    int fused_query_end     = 0;
```

Comment:
Also @edawson is there a latent bug here? Compiler was throwing a uninitialized variable error here, which could happen in some cases. Is initializing to 0 a reasonable change or do we need to set the variable previously like is done on line 168?

Reply:
I think this should be fine - it doesn't look like these get used or set elsewhere (besides line 168). It might be good to set them to INT_MAX since zero is a valid value, but otherwise I think this should work.

Reply:
Optional: I would prefer `std::numeric_limits<position_in_read_t>::max()` over the MAX macros, that way it will still work in case we ever change the underlying type. (You need to add `#include <limits>`.)

Reply:
That's a good point. Changed to use numeric_limits in the latest version